client/app/components/text-input.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+
+export default class TextInputComponent extends Component {
+  maxLength = this.args.maxLength || '250';
+
+  showCounter = this.args.showCounter || false;
```

Comment:
Just FYI, if someone should decide that the default behavior of this component is to show counter (`true`), and someone on the consuming end passes `false`, it will prefer `true`.

Reply:
😱  😮  omg,  thanks for pointing that out!